main.tf

```diff
@@ -1,3 +1,6 @@
+data "google_cloud_identity_groups" "groups" {
+  parent = var.parent
```

Comment:
I think we forgot about: https://github.com/getindata/terraform-snowflake-schema/blob/main/main.tf#LL99C3-L99C38

Reply:
Hmm - you're right ... Can we omit `parent` somehow and pass null ?